docs/configuration.md

```diff
@@ -0,0 +1,97 @@
+# Configuration
```

Comment:
I think having the configuration file separate might generate confusion. This is only valid for the setup using the example docker-compose.yml, but do not explain all of the possible configuration values for all services.

Reply:
After a bit of thinking better to rename this to `deploy-configuration.md` as it spans across all services. The `configuration.md` should be reserved for K-Link site specific configuration